### PR TITLE
CY-2099: status reporter token

### PR DESCRIPTION
**Pull request by tyacbovi**
- Fix circleci test to use the same 'encoding_alphabet'
- Fix not generating admin token in community

### CODE REVIEW

File: cfy_manager/components/restservice/restservice.py [hunk 1]
```diff
@@ -306,10 +306,10 @@ def _validate_cluster_join(self):
 
         certs = config[SSL_INPUTS]
         internal_certs_provided = (
-            certs['internal_cert_path'] and certs['internal_key_path']
+                certs['internal_cert_path'] and certs['internal_key_path']
```
Comment (geokala) [inline cfy_manager/components/restservice/restservice.py]: This doesn't need an extra level of indentation

File: .circleci/cluster/manager1_config.yaml [hunk 2]
```diff
@@ -50,5 +50,8 @@ ssl_inputs:
   external_key_path: /etc/cloudify/external_key.pem
   external_ca_cert_path: /etc/cloudify/ca.pem
 
+flask_security:
+  encoding_alphabet: 5TNRm2piPIOXn6FVAhcJeUSjswEHYbg
```
Comment (geokala) [inline .circleci/cluster/manager1_config.yaml]: Why are we forcing this to one value for CI?

File: cfy_manager/components/restservice/restservice.py (hunk 1 above)
Comment (tyacbovi) [reply cfy_manager/components/restservice/restservice.py]: Pycharm is a sneaky one, will remove

File: .circleci/cluster/manager1_config.yaml (hunk 2 above)
Comment (tyacbovi) [reply .circleci/cluster/manager1_config.yaml]: Just for keeping it simple and stable

Comment (geokala) [reply .circleci/cluster/manager1_config.yaml]: We've actually managed to cause ourselves a variety of hidden bugs by having values that don't change in images before, so it'd be better if we didn't do this if we can avoid it.

File: cfy_manager/components/restservice/restservice.py [hunk 3]
```diff
@@ -158,6 +158,7 @@ def _get_flask_security(self):
         # If we're recreating the DB, or if there's no previous security
         # config file, just use the config that was generated
         if config[CLEAN_DB] or not exists(REST_SECURITY_CONFIG_PATH):
+            logger.error('NOOOOO')
```
Comment (geokala) [inline cfy_manager/components/restservice/restservice.py]: I approve of this message.

**Pull request merged**